src/python/grpcio/grpc/_cython/_cygrpc/aio/server.pyx.pxi

```diff
@@ -541,7 +584,7 @@ cdef CallbackFailureHandler SERVER_SHUTDOWN_FAILURE_HANDLER = CallbackFailureHan
 cdef class AioServer:
 
     def __init__(self, loop, thread_pool, generic_handlers, interceptors,
-                 options, maximum_concurrent_rpcs, compression):
+                 options, maximum_concurrent_rpcs):
```

Comment:
Correct me if I'm wrong but this is the way on the synchronous server or enabling the compression, what's the rationale on not giving support to this?

Reply:
It is still supported. I found the compression can be specified in options (aka. channel argument), so I merged the compression channel argument in `Server.__init__` from `_server.py`. A new test case added `test_server_default_compression_algorithm`.

Also, there are three tuning options for compression: 1. set the default algorithm; 2. set the desired compression level (see [doc](https://github.com/grpc/grpc/blob/master/doc/compression_cookbook.md)); 3. supported algorithms. I hope we can promote them to first class API in future. Do you think they are helpful?

Reply:
Thanks.

By chance the server that we are targeting for test the `aio` module has the compression enabled :), but they are basically using the `compression` parameter in the server side, beyond that I dio not have more experience about how useful is having all of these methods for enabling, tunning the compression. So I will give you feedback once I have it :)

Something that has been asked is when the `experimental` flag will be removed for this feature. Im wondering too what would need to happen for not having it under the experimental flag. Any idea?